spec/features/register_command_spec.rb

```diff
@@ -229,10 +229,209 @@
+        md_rec2 = get_metadata_record(file_path2, md_dir)
+        expect(md_rec2.checksums['sha1']).to eq '013d5696728f086b8d2424b14beebc2695f926f7'
+        expect(last_command_started).to have_exit_status(0)
+      end
+    end
+
+    context 'register multiple files with multiple checks from multiple manifests' do
+      let!(:file_path2) { create_test_file(dir: path_dir, name: 'another_file', content: 'more content') }
+      let!(:manifest_path) { create_test_file(dir: path_dir, name: 'manifest1.txt',
+          content: "e8241901910dac399e9fcd5fb5661e6923a0ce0a   #{file_path}\n" +
+                   "013d5696728f086b8d2424b14beebc2695f926f7   #{file_path2}\n") }
+
+      let!(:manifest_path2) { create_test_file(dir: path_dir, name: 'manifest2.txt',
+          content: "9f753c302ffa359ddb9a93fe979d6de1   #{file_path}\n" +
+                   "ce6320a83ead310ae30d43ae0f338bcc   #{file_path2}\n") }
+
+      before do
+        run_command_and_stop("longleaf register -c #{config_path} -m sha1:#{manifest_path} md5:#{manifest_path2}", fail_on_error: false)
+      end
+
+      it 'registers the files with digests' do
+        expect(last_command_started).to have_output(/SUCCESS register #{file_path}/)
+        md_rec1 = get_metadata_record(file_path, md_dir)
+        expect(md_rec1.checksums['sha1']).to eq 'e8241901910dac399e9fcd5fb5661e6923a0ce0a'
+        expect(md_rec1.checksums['md5']).to eq '9f753c302ffa359ddb9a93fe979d6de1'
+        expect(last_command_started).to have_output(/SUCCESS register #{file_path2}/)
+        md_rec2 = get_metadata_record(file_path2, md_dir)
+        expect(md_rec2.checksums['sha1']).to eq '013d5696728f086b8d2424b14beebc2695f926f7'
+        expect(md_rec2.checksums['md5']).to eq 'ce6320a83ead310ae30d43ae0f338bcc'
+
+        expect(last_command_started).to have_exit_status(0)
+      end
+    end
+
+    context 'register multiple files with multiple checks from combined manifest' do
+      let!(:file_path2) { create_test_file(dir: path_dir, name: 'another_file', content: 'more content') }
+      let!(:manifest_path) { create_test_file(dir: path_dir, name: 'manifest1.txt',
+          content: "sha1:\n" +
+                   "e8241901910dac399e9fcd5fb5661e6923a0ce0a   #{file_path}\n" +
+                   "013d5696728f086b8d2424b14beebc2695f926f7   #{file_path2}\n" +
+                   "md5:\n" +
+                   "9f753c302ffa359ddb9a93fe979d6de1   #{file_path}"
+                   ) }
+
+      before do
+        run_command_and_stop("longleaf register -c #{config_path} -m #{manifest_path}", fail_on_error: false)
+      end
+
+      it 'registers the files with digests' do
+        expect(last_command_started).to have_output(/SUCCESS register #{file_path}/)
+        md_rec1 = get_metadata_record(file_path, md_dir)
+        expect(md_rec1.checksums['sha1']).to eq 'e8241901910dac399e9fcd5fb5661e6923a0ce0a'
+        expect(md_rec1.checksums['md5']).to eq '9f753c302ffa359ddb9a93fe979d6de1'
```

Comment:
This all looks good! It does seem like you might be able to make these tests a little more succinct with a helper method for comparing checksums in metadata files